src/Simplify.cpp

```diff
@@ -177,37 +175,69 @@ class Simplify : public IRMutator {
                 *max_val = b.second;
                 return true;
             }
+        } else if (const Broadcast *b = e.as<Broadcast>()) {
+            return const_int_bounds(b->value, min_val, max_val);
+        } else if (const Max *max = e.as<Max>()) {
+            int64_t min_a, min_b, max_a, max_b;
+            if (const Min *min = max->a.as<Min>()) {
```

Comment:
Why isn't there a mirror case for max->b? If we canonicalize these such that this is the only case we need to deal with, can you add a comment noting that?